specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2020-10-01/examples/BackupResourceEncryptionConfig_Get.json

```diff
@@ -0,0 +1,25 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "rishgrp",
+    "vaultName": "rishTestVault",
+    "api-version": "2020-10-01"
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rishgrp/providers/Microsoft.RecoveryServicesBVTD2/vaults/rishTestVault/backupEncryptionConfigs/backupResourceEncryptionConfig",
+        "name": "backupResourceEncryptionConfig",
+        "type": "Microsoft.RecoveryServices/vaults/backupEncryptionConfigs",
+        "properties": {
```

Comment:
>properties [](start = 9, length = 10)

We have this contract documented for CMK scenarios: [https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-contracts.md#customer-managed-key-encryption](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-contracts.md#customer-managed-key-encryption). I see the structure is different than yours. Would the documented format be applicable to your situation?

Reply:
No. The structure is specific to Backup service.

Reply:
From customer perspective I should be able to setup CMK for multiple data types why do we have differentiation?

Reply:
Yes - the settings are common for all the data sources. What I meant is the format given in the above API is specific to Backup Service.

Reply:
Discussed with Jack from BYOK team - there was guidance on contract, but wasn't mandated owing to JEDI contract and timeline. We will continue to use the current contract, In the next release where we target user identities, we will work with ARM team and Jack to ensure our contracts are in sync with recommendation. The recommendation above also needs to be updates to be in sync with BYOK recommendation.